docs/tutorials/byproduct_recovery_determination.ipynb

```diff
@@ -0,0 +1,706 @@
+      "  53  2.5124072e+05 7.02e-09 4.52e-07  -5.7 4.76e-01    -  1.00e+00 1.00e+00h  1\n",
+      "  54  2.5124071e+05 2.91e-11 7.09e-11  -8.6 5.98e-03    -  1.00e+00 1.00e+00h  1\n",
+      "\n",
+      "Number of Iterations....: 54\n",
+      "\n",
+      "                                   (scaled)                 (unscaled)\n",
+      "Objective...............:   2.5124071143455856e+02    2.5124071143455850e+05\n",
+      "Dual infeasibility......:   7.0857863511826187e-11    7.0857863511826174e-08\n",
+      "Constraint violation....:   7.2759576141834259e-12    2.9103830456733704e-11\n",
+      "Complementarity.........:   2.5060987127300026e-09    2.5060987127300018e-06\n",
+      "Overall NLP error.......:   2.5060987127300026e-09    2.5060987127300018e-06\n",
+      "\n",
+      "\n",
+      "Number of objective function evaluations             = 102\n",
+      "Number of objective gradient evaluations             = 30\n",
+      "Number of equality constraint evaluations            = 102\n",
+      "Number of inequality constraint evaluations          = 102\n",
+      "Number of equality constraint Jacobian evaluations   = 56\n",
+      "Number of inequality constraint Jacobian evaluations = 56\n",
+      "Number of Lagrangian Hessian evaluations             = 54\n",
+      "Total CPU secs in IPOPT (w/o function evaluations)   =      0.032\n",
+      "Total CPU secs in NLP function evaluations           =      0.003\n",
+      "\n",
+      "EXIT: Optimal Solution Found.\n"
+     ]
+    }
+   ],
+   "source": [
+    "m = build_model()\n",
+    "add_costing(m)\n",
+    "initialize_model(m)\n",
+    "solve_model(m)\n",
+    "\n",
+    "unfix_opt_variables(m)\n",
+    "add_product_constraints(m, Li_recovery_bound=0.95, Co_recovery_bound=0.635)\n",
+    "add_objective(m)\n",
+    "set_scaling(m)\n",
+    "scaling = TransformationFactory(\"core.scale_model\")\n",
+    "scaled_model = scaling.create_using(m, rename=False)\n",
+    "solve_model(scaled_model)\n",
+    "# Propagate results back to unscaled model\n",
+    "scaling.propagate_solution(scaled_model, m)\n",
+    "\n",
+    "# Store results for later tests\n",
+    "total_annualized_cost = value(m.fs.costing.total_annualized_cost)\n",
+    "\n",
+    "# recovery mass flow rate kg/hr\n",
+    "Li_recovery_mass = value(m.fs.stage3.permeate_outlet.flow_vol[0]) * value(\n",
+    "    m.fs.stage3.permeate_outlet.conc_mass_solute[0, \"Li\"]\n",
+    ")\n",
+    "Co_recovery_mass = value(m.fs.stage1.retentate_outlet.flow_vol[0]) * value(\n",
+    "    m.fs.stage1.retentate_outlet.conc_mass_solute[0, \"Co\"]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f9e6bd8b-87da-4467-99a4-36e651a83ba1",
+   "metadata": {},
+   "source": [
+    "## __Step 3: Test the Byproduct Recovery Framework__\n",
+    "\n",
+    "With the mass balance from previous step, the next task is to calculate the __cost__ and evaluate __economic feasibility__. \n",
```

Comment:
"With the mass balance" is a bit confusing, can we say something like "now that the model is solved" instead?